drape/overlay_tree.cpp

```diff
@@ -79,6 +117,12 @@ void OverlayTree::Add(ref_ptr<OverlayHandle> handle, bool isTransparent)
 
 void OverlayTree::EndOverlayPlacing()
 {
+  HandleComparator comparator;
+  sort(m_handles.begin(), m_handles.end(), bind(&HandleComparator::operator(), &comparator, _1, _2));
+  for (auto const & handle : m_handles)
+    InsertHandle(handle.first, handle.second);
+  m_handles.clear();
```

Comment:
Do you need to return physical block of memory to the system here? Or m_handles will be reused later?